doc/versions.rst

```diff
@@ -10,3 +10,7 @@ Web-based documentation is available for versions listed below:
 * `Nilearn 0.9.2 documentation <http://nilearn.github.io/0.9.2/>`_
 * `Nilearn 0.9.1 documentation <http://nilearn.github.io/0.9.1/>`_
 * `Nilearn 0.9.0 documentation <http://nilearn.github.io/0.9.0/>`_
+* `Nilearn 0.8.1 documentation <http://nilearn.github.io/0.8.1/>`_
+* `Nilearn 0.8.0 documentation <http://nilearn.github.io/0.8.0/>`_
```

Comment:
This one is returning 404

Reply:
http://nilearn.github.io/0.8.0/

Reply:
> This one is returning 404

thanks for checking

I had borked the 'back up' of this version: should be fixed soon

Reply:
works now